data/flows/patterns.yml

```diff
@@ -25,11 +25,13 @@ flows:
     steps:
       - action: action_trigger_chitchat
 
+  # using chitchat here so that intentless is used for better testability
```

Comment:
does this mean that some of the failing tests are now passing? if yes, should we move them?